app/koi-client/src/component-presentation/MobileLayout.tsx

```diff
@@ -1,6 +1,5 @@
 import React, { CSSProperties } from 'react';
-import { css } from '@linaria/core';
-import { useMediaQuery } from 'react-responsive';
+import { styled } from '@linaria/react';
```

Comment:
```suggestion
import styled from '@emotion/styled';
```

emotion으로 수정 부탁드립니다~!